docs/analytics.rst

```diff
@@ -118,3 +118,37 @@ When a banner is dismissed:
         'data-banner-dismissal': '1',
         'event': 'in-page-interaction'
     });
+
+
+When doing a/b tests configure something like the following.
+
+.. code-block:: javascript
+    if(href.indexOf('v=a') !== -1) {
+        window.dataLayer.push({
+            'data-ex-variant': 'de-page',
+            'data-ex-name': 'Berlin-Campaign-Landing-Page'
+        });
+    } else if (href.indexOf('v=b') !== -1) {
+        window.dataLayer.push({
+            'data-ex-variant': 'campaign-page',
+            'data-ex-name': 'Berlin-Campaign-Landing-Page'
+        });
+    }
+
+
+Some notes on how this looks in GA
+----------------------------------
+
+``data-link-type="button"`` and ``data-link-name=""`` trigger a generic link
+click with the following structure:
+
+    Event Category: {{page ID}} Interactions
```

Comment:
You need a [line block](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#line-blocks) here. e.g.:

```rst
| Event Category: {{page ID}} Interactions
| Event Action: {{data-link-type}} click
| Event Label: {{data-link-name}}
```

That will preserve the linebreaks.